### PR TITLE
[6799] budegting proposal back button

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

**Review by Rineee — COMMENTED** [review]
Supercool, that totally works!

But Caro wants another wording for the back button, see story comments.

**Comment by fuzzylogic2000** [comment]
> Supercool, that totally works!
> 
> But Caro wants another wording for the back button, see story comments.

Oh, that comment came too late for me. Do you think, we should keep the name in the context or should I remove everything again? Yeah, writing that I already know it is removing it. :(

**Comment by Rineee** [comment]
> > Supercool, that totally works!
> > But Caro wants another wording for the back button, see story comments.
> 
> Oh, that comment came too late for me. Do you think, we should keep the name in the context or should I remove everything again? Yeah, writing that I already know it is removing it. :(

Yeah, sorry, I guess we dont need it anymore then :/

File: meinberlin/apps/maptopicprio/templates/meinberlin_maptopicprio/maptopic_detail.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
                 <li>
                     <a href="{{ module.get_detail_url }}">
                         <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                        {% trans 'go back' %}
+                        {% trans 'list' %}
```
Comment (Rineee) [inline meinberlin/apps/maptopicprio/templates/meinberlin_maptopicprio/maptopic_detail.html:23]: @fuzzylogic2000 translate :nerd_face:

File: meinberlin/apps/topicprio/templates/meinberlin_topicprio/topic_detail.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
                 <li>
                     <a href="{{ module.get_detail_url }}">
                         <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                        {% trans 'go back' %}
+                        {% trans 'list' %}
```
Comment (Rineee) [inline meinberlin/apps/topicprio/templates/meinberlin_topicprio/topic_detail.html:13]: translate

**Review by Rineee — APPROVED** [review]
Yes, perfect!

**Pull request merged**